### DISCUSSION TITLE
Monitor Interface missing on reboot

### DISCUSSION THREAD

**Discussion by BSUNetSec** [Security-Onion-Solutions/securityonion#12273] [Unsupported Versions]
Greetings,

I have noticed that on reboot I will lose the monitor interface on our sensors.  I can add them back with so-monitor-add but I am trying to determine why I will lose it on reboot.  Has anyone else run into this issue?

Running 2.4.40

Thanks

**Comment by cm-ops** [comment]
Can you provide some more background? What was your install type ISO v. network, OS, physical hardware or VM, etc.

**Reply by BSUNetSec** [reply]
Distributed Install
1 Manager/Search - Manual Install of Rocky Linux and Manual install of SO
7 Forward Nodes - Each node running ubuntu 22.04 and a 10Gb BCM57416

All boxes are physical systems.  Issue started after rebuilding from 2.3 to 2.4.  When I reboot the sensors, the monitoring interface unbinds from the bond0 interface and I have to add them back using so-monitor-add, after which it works like intended.  Not sure if I need to add the interface to a config file somewhere to fix this or not.
Thanks

**Reply by cm-ops** [reply]
What does this show? `cat /proc/net/bonding/bond0` Is there a reason you did not use the ISO and the supported OS? https://docs.securityonion.net/en/2.4/os.html#supported
